quadrat/sass/templates/_meta.scss

```diff
@@ -28,6 +28,9 @@
 			background: transparent;
 			-webkit-mask-image: none;
 			mask-image: none;
+			height: 0;
```

Comment:
Could we just use `content: none;` here and remove the rest of CSS specific to the `:before` element?

Reply:
I think this makes a good case for making the post meta rules opt in in blockbase, as suggested in https://github.com/Automattic/themes/issues/4704